incubator/binding-grpc-kafka/src/main/java/io/aklivity/zilla/runtime/binding/grpc/kafka/internal/config/GrpcKafkaConditionConfigAdapter.java

```diff
@@ -63,7 +63,7 @@ public JsonObject adaptToJson(
         if (condition.service != null)
         {
             String method = condition.service;
-            method = condition.method != null ? method + "/" + condition.method : method;
+            method = condition.method != null ? String.format("%s/%s", method, condition.method) : method;
```

Comment:
Shouldn't we have `/*` on the end when the `method` is `null` instead of just the `service` by itself?

Reply:
if the user has specified `method: test` then it doesn't expect /* I think no?

Reply:
In the schema, we only have `method` now, but we're parsing the value into `service` and `method`, where the `service` part is required. 

However, having the syntax `method: test` to mean the service is `test` and the method is `*` is too confusing, agree?

Recommend we change the pattern to require `<Service>/<Method>` where `<Method>` can be `*`, so that `method: test/*` means service `test` and any method.